### DISCUSSION TITLE
Auto restart

### DISCUSSION THREAD

**Discussion by FlemmingJPetersen** [Rem0o/FanControl.Releases#2085] [Ideas]
Is it possible to get an auto restart option whenever the a problem like this occurs?
![Capture](https://github.com/Rem0o/FanControl.Releases/assets/12712283/b0ffeea5-3123-4485-8c1c-ae1f6690a6d3)

**Comment by Rem0o** [comment]
That's an unexpected error. Can be a bug, can be something else. Without knowing what it is exactly, restarting is most likely a swing and a miss. That's not how these error should be handled.   

The error log must be checked.

**Reply by FlemmingJPetersen** [reply]
I my case I know what the problem and a restart is currently the only way to fix it.

Sometimes the HWInfo fails, which for my case, stops all fans.
I have writte a powershell script the fixes it for me, but it would be nice FanControl could handle it.
